src/server/scripts/Outland/GruulsLair/boss_gruul.cpp

```diff
@@ -114,6 +114,10 @@ class boss_gruul : public CreatureScript
                 case EVENT_HURTFUL_STRIKE:
                     if (Unit* target = SelectTarget(SELECT_TARGET_TOPAGGRO, 1, 5.0f))
                         me->CastSpell(target, SPELL_HURTFUL_STRIKE, false);
+                    else if (me->GetVictim())                                           // If no target was found for Hurtful Strike, target the MT
```

Comment:
No need for the `if (...)` since if there's no victim, the boss will reset, avoiding an unnecessary check

Reply:
same as above, @55Honey